Google.Api.Generator.Tests/ProtoTests/VoidReturn/Testing.VoidReturn.GeneratedSnippets/snippet_metadata_testing.voidreturn.json

```diff
@@ -105,6 +105,55 @@
         }
       ]
     },
+    {
+      "regionTag": "unknown_generated_VoidReturn_VoidMethod_async",
```

Comment:
Is the region tag expected to be unique? If so, we'll need some sort of disambiguation.